modules/boost.assert/1.83.0.bcr.1/presubmit.yml

```diff
@@ -0,0 +1,16 @@
+matrix:
+  platform:
```

Comment:
Since this is a header only lib, we could also make use of the `--process_headers_in_dependencies` feature -> https://github.com/bazelbuild/bazel-central-registry/blob/main/modules/boost.detail/1.83.0/presubmit.yml -> will restrict us to Bazel >=7.2.1

Reply:
updated

Reply:
erm, this actually doesn't do anything on it's own afaict. you also have to pass `--features=parse_headers`, is that true @fmeum ? should we dedup that in bazel?

Reply:
Have also a look at the BUILD file of boost.detail

Reply:
pushed